common/http.hpp

```diff
@@ -533,6 +534,59 @@ class Server : public events::EventLoopObject {
 #endif // MENDER_USE_BOOST_BEAST
 };
 
+class ExponentialBackoff {
```

Comment:
Might be worth it to change this into a template so the unit doesn't always have to be milliseconds.

Reply:
Makes sense. But since all coarser timers are already compatible with it, and we don't need more precision, maybe we can keep it non-templated just to save code space?

Reply:
Yup, I didn't realize `chrono::` took care of the unit conversions.